taxonomies/food/ingredients.txt

```diff
@@ -5172,6 +5175,7 @@ it:derivati del siero di latte, la preparazione del siero di latte
 nl:Wei
 pl:produkt serwatkowy
 sv:vassleprodukt
+nova:en:4
```

Comment:
We should put "whey" directly in nova 4:
"Substances only found in ultra-processed products include some directly extracted from
foods, such as casein, lactose, whey, and gluten, and some derived from further processing
of food constituents, such as hydrogenated or interesterified oils, hydrolysed proteins, soy
protein isolate, maltodextrin, invert sugar and high fructose corn syrup. Classes of additive
only found in ultra-processed products include dyes and other colours, colour stabilisers,
flavours, flavour enhancers, non-sugar sweeteners, and processing aids such as
carbonating, firming, bulking and anti-bulking, de-foaming, anti-caking and glazing agents,
emulsifiers, sequestrants and humectants."
https://archive.wphna.org/wp-content/uploads/2016/01/WN-2016-7-1-3-28-38-Monteiro-Cannon-Levy-et-al-NOVA.pdf

Reply:
ok done